teb_local_planner/include/teb_local_planner/teb_config.h

```diff
@@ -275,9 +275,9 @@ class TebConfig
     robot.max_vel_y = 0.0;
     robot.max_vel_theta = 0.3;
     robot.base_max_vel_x = robot.max_vel_x;
-    robot.base_max_vel_x_backwards = robot.base_max_vel_x_backwards;
-    robot.base_max_vel_y = robot.base_max_vel_y;
-    robot.base_max_vel_theta = robot.base_max_vel_theta;
+    robot.base_max_vel_x_backwards = robot.max_vel_x_backwards;
+    robot.base_max_vel_y = robot.max_vel_y;
+    robot.base_max_vel_theta = robot.max_vel_theta;
```

Comment:
Conceptually I think we should be setting the "base velocities" in our config and the other speed limited velocities are the variable ones. Changing this would we config-breaking though